PowerVCF.psm1

```diff
@@ -47,22 +47,22 @@ if ($PSEdition -eq 'Desktop') {
 
 ####  Do not modify anything below this line. All user variables are in the accompanying JSON files #####
 
-Function Connect-VCFManager
```

Comment:
Why rename ?

Reply:
The rename is because you are not "connecting" to SDDC manager in the same way you connect to vCenter. You are requesting an API token. So Request-VCFToken makes more sense

Reply:
And it is aliased for backwards compatibility until deprecated.

Reply:
i prefer connect... (a lot of module use connect to get token...)

Reply:
Final say went to original author of the function 🤣